wmn-data.json

```diff
@@ -685,6 +685,18 @@
         "m_code" : 200,
         "known" : ["treasdior", "Jacin19"],
         "cat" : "gaming",
+        "valid" : true
+       },
+       {
+        "name" : "Bikemap",
+        "uri_check" : "https://www.bikemap.net/en/u/{account}/routes/created/",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "- 🚲 Bikemap</title>",
+        "m_string" : "<title>Page not found - Error 404 ",
+        "m_code" : 404,
+        "known" : ["mike", "greg"],
+        "cat" : "hobby",

```

Comment:
Can you change this to `health` since this is about exercising?

Reply:
@WebBreacher No problem, will do